guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -48,8 +48,14 @@ endif::[]
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *Register Host*.
-. Enter the details for how you want the registered host to be configured.
+. Enter the details for how you want the registered host to be configured. Note:
+* If you select a host group from the *Host Group* list, the *Operating system*
 ifdef::katello,satellite,orcharhino[]
+, *Activation Keys*
+endif::[]
+and *Lifecycle environment* fields inherit their values from the host group.
+ifdef::katello,satellite,orcharhino[]
+* A Capsule behind a load balancer takes precedence over the Capsule selected in this UI as the host’s content source.
```

Comment:
```suggestion
* A {SmartProxy} behind a load balancer takes precedence over the {SmartProxy} selected in the {ProjectWebUI} as the content source of the host."
```
Attributes to avoid branding issues and minor rewording to avoid possessive apostrophe.